### ISSUE TITLE
Support for React.Component class and constructor function snippets

### ISSUE THREAD

**Issue by numaanashraf** [webbushka/atom-react-snippets#3]
Any plans to add the following snippets?

```
class TodoItem extends React.Component {
   // ...
}
```

and 

```
constructor(props) {
    super(props)
    // ...
}
```

**Comment by webbushka** [comment]
Yes! I am planning to add them very soon, or will accept a pull request.

**Comment by numaanashraf** [comment]
Awesome! PR coming up!

**Comment by webbushka** [comment]
This was fixed by #4
